src/components/navigation/ShareModal.tsx

```diff
@@ -69,57 +75,78 @@ export default function ShareModal(props) {
   ]
 
   return (
-    <Modal isOpen={open} onRequestClose={onClose} style={ModalStyles}>
-      <StyledH2>Teile den Inhalt!</StyledH2>
+    <StyledModal isOpen={open} onRequestClose={onClose}>
+      <StyledH2>Weitergeben!</StyledH2>
       <div>
         <ShareInput
           ref={shareInputRef}
           onFocus={e => e.target.select()}
           defaultValue={url}
         />{' '}
         {document.queryCommandSupported('copy') && (
-          <Button onClick={copyToClipboard}>
-            <FontAwesomeIcon icon={faCopy} /> Kopieren
-          </Button>
+          <>
+            {copySuccess !== '' && <Gray>{copySuccess}&nbsp;</Gray>}
+            <br />
+            <Button onClick={copyToClipboard}>
+              <FontAwesomeIcon icon={faCopy} /> Kopieren
+            </Button>
+          </>
         )}{' '}
-        <br />
-        <Gray>{copySuccess}&nbsp;</Gray>
         <CloseButton onClick={onClose} title="Close">
           <FontAwesomeIcon icon={faTimes} size="lg" />
         </CloseButton>
-        <div>{buildButtons(socialShare)}</div>
-        <br />
-        <div>{buildButtons(lmsShare)}</div>
+        <ButtonWrapper>{buildButtons(socialShare)}</ButtonWrapper>
+        <ButtonWrapper>{buildButtons(lmsShare)}</ButtonWrapper>
       </div>
-    </Modal>
+    </StyledModal>
   )
 }
 
-const ModalStyles = {
-  content: {
-    top: '50%',
-    left: '50%',
-    right: 'auto',
-    bottom: 'auto',
-    marginRight: '-50%',
-    transform: 'translate(-50%, -50%)',
-    width: '480px',
-    borderRadius: '17px',
-    maxWidth: '85%'
-  }
+function buildButtons(list) {
+  return list.map(entry => (
+    <Button href={entry.href} key={entry.title}>
+      <FontAwesomeIcon icon={entry.icon} /> {entry.title}
+    </Button>
+  ))
 }
 
-const StyledH2 = styled.h2`
-  font-weight: 700;
-  line-height: 1.35;
-  letter-spacing: -0.01em;
+export const StyledModal = styled(Modal)`
```

Comment:
cool that this works. just to document this behaviour: add a small comment here pointing out that we are overriding the content styles

Reply:
Jup, thought about adding it to the readme as well, but it's an special case.
There seems to be a way to change all modal styles with styled components, it's okay but not super clean.

If we ever need it or if you prefer this to be the standard way: https://github.com/reactjs/react-modal/issues/627#issuecomment-578814799